src/chrome/content/rules/Bilibili.com.xml

```diff
@@ -2,15 +2,13 @@
 	Relate ruleset:
 		BiliGame.xml
 
-	CORS:
+	CORS: https://github.com/EFForg/https-everywhere/pull/7495#discussion_r95588380
 		- www.bilibili.com
-			( detail: https://github.com/EFForg/https-everywhere/pull/7495#discussion_r95588380 )
+			( No functional breakage. Test: https://bangumi.bilibili.com/22/ )
 		- api.bilibili.com
-			( test: http://h.bilibili.com/dy\d+ )
-		- comment.bilibili.com
-			( test: https://www.bilibili.com/video/av\d+/ )
+			( Functional breakage. Test: http://h.bilibili.com/dy\d+ )
```

Comment:
Please list the issue specifically, something likes "Comments are not displaying properly"